typescript/sdk/src/providers/MultiProvider.ts

```diff
@@ -346,11 +394,8 @@ export class MultiProvider<MetaExt = {}> extends ChainMetadataManager<MetaExt> {
     const confirmations =
       this.getChainMetadata(chainNameOrId).blocks?.confirmations ?? 1;
     const response = await tx;
-    const txUrl = this.tryGetExplorerTxUrl(chainNameOrId, response);
```

Comment:
why remove this? would be great if we can still link to the zksync block explorers

Reply:
Missed it, will bring it back